### DISCUSSION TITLE
Trascrizione di adozione dall'estero: dati provvedimento di origine (Trascr_018 - Trascr_020)

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#1041] [Supporto servizi cooperativi] [answered]
I due casi d'uso:
_Trascr_018 - Trascrizione di nascita: Trascrizione provvedimento straniero di adozione richiesta da un privato
Trascr_020 - Trascrizione di nascita: Trascrizione di provvedimento straniero adozione di minorenne o maggiorenne in casi particolari richiesta da un privato_

fanno entrambi riferimento a situazione nella quale sia il soggetto privato a richiedere all'usc la trascrizione di un provvedimento di adozione prodotto all'estero.

Nel caso "Trascr_018", in riferimento al provvedimento (estero) originario sono presenti le due sezioni (obbligatorie):

![image](https://github.com/user-attachments/assets/909eeea9-520e-4728-ba04-ea29a50d9747)

che - immaginiamo - debbano contenere, rispettivamente:
a) gli estremi del provvedimento estero da trascrivere
b) i dati identificativi dell'autorità estera che abbia emesso il provvedimento (estero) originario.

Nel caso "Trascr_020", in riferimento al provvedimento (estero) originario è presente la sola sezione (obbligatoria):

![image](https://github.com/user-attachments/assets/c966ebb9-41b4-4b6d-acf4-220d01eed469)

che - immaginiamo - debba contenere gli estremi del provvedimento estero da trascrivere.

Anzitutto chiediamo: è corretta la nostra ipotesi sulla valorizzazione delle sezioni di cui sopra?

Chiarito questo, chiediamo: esiste un motivo specifico per il quale nel secondo caso (Trascr_020) non sono richiesti i dati identificativi dell'autorità estera che abbia emesso il provvedimento (estero) originario.
Oppure è prevista specifica integrazione futura del caso d'uso (Trascr_020)?

La richiesta nasce dall'esigenza di ottimizzare la pianificazione delle integrazioni necessarie al gestionale locale al fine della gestione completa dei caso d'uso complessivamente afferenti ai procedimenti di adozione.

Rimaniamo in attesa di vostro riscontro.

**Comment by asqueglia** [comment]
Buongiorno @antonioluberto, trattandosi dello stesso caso d’uso discusso in #944 , anche su questo punto chiederemo un approfondimento ad ANSUCA

**Comment by asqueglia** [comment, marked as answer]
Buongiorno @antonioluberto, dall'ulteriore approfondimento effettuato con ANUSCA in data odierna, si conferma che verrà inserita  la sezione Autorità estera obbligatoria; per cui verrà presa in carico la segnalazione che trasformiamo quindi in una issue

**Comment by asqueglia** [comment]
Buon pomeriggio, 
da un ulteriore approfondimento dei casi citati abbiamo appurato che sulla tabella di decodifica la descrizione del caso d'uso Teascr_018 è errata infatti è diversa da quanto presente sulla Web App.
il caso d'uso Trascr_018 è infatti: Trascrizione provvedimento straniero di adozione ed utilizza la formula 193; la descrizione completa dovrebbe essere:
Trascrizione provvedimento straniero di adozione dichiarato efficace in Italia dal Tribunale dei Minori e trasmesso al comune di residenza dei genitori adottivi
Sembra quindi che l'errore sia sulla decodifica dec_use_case e che il caso sia stato configurato correttamente richiedendo l'autorità mittente che richiede la trascrizione.
 
Trascr_020 invece è la Trascrizione di provvedimento straniero di adozione di minorenne o maggiorenne in casi particolari richiesta da un privato ed utilizza la formula 192.
 
I due casi sono molto simili, ed attualmente entrambi richiedono obbligatoriamente l' "autorità estera" che è presente nella sezione "Atto da Trascrivere".
 
La sezione "Autorità mittente" è presente solo in trascr_018 e non in trascr_020 perchè ciò che cambia è appunto il richiedente trascr_018 è una trascrizione richiesta da ente mentre trascr_020 è richiesta da privato.

Ci scusiamo per l'errore nella descrizione del caso d'uso sulla tabella di decodifica, che provvederemo a correggere con il prossimo rilascio

**Comment by antonioluberto** [comment]
Buongiorno.
A questo punto, Trascr_018 deve essere usato solo nel caso in cui il decreto di adozione sia **trasmesso**  al comune di residenza dei genitori adottivi **direttamente** dal Tribunale?
E a quale caso d'uso dovremo are riferimento laddove la trascrizione del decreto straniero di adozione sia richiesta da privato?

**Comment by asqueglia** [comment]
Come detto i due casi d'uso da prendere in  considerazione sono rispettivamente  il Trascr_018 e il Trascr_020. Chiudiamo pertanto la discussion
